docs/docs/Contributing/contributing-templates.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Contribute templates
+slug: /contributing-templates
+---
+
+Follow these best practices when submitting a template to Langflow.
+
+For template formatting examples, see the Langflow repository's [starter_projects](https://github.com/langflow-ai/langflow/tree/main/src/backend/base/langflow/initial_setup/starter_projects) folder.
+
+## Create a PR to submit your template
+
+Follow these steps to submit your template:
+
+1. Fork the Langflow repository on GitHub.
+2. Add your `template.json` file to the `src/backend/base/langflow/initial_setup/starter_projects` folder in your fork.
+3. Create a Pull Request from your fork to the main Langflow repository.
+4. Include a screenshot of your template in the PR.
```

Comment:
You should add a link to the required items for template submission somewhere in this procedure. People may skip that section otherwise.